quipucords/tests/scanner/network/processing/test_dmi.py

```diff
@@ -31,12 +37,13 @@ def test_success_case(self):
 
     def test_invalid_uuid_case(self):
         """Test dmi_system_uuid too long."""
-        # stdout_lines looks like ['a', 'b', 'c']
-        dependencies = {"internal_dmi_system_uuid": ansible_result(f"{'a' * 37}\nb\nc")}
+        # stdout_lines looks like ['a', 'b', 'ccccccccccccccccccccccccccccccccccccc']
+        dependencies = {"internal_dmi_system_uuid": ansible_result(f"a\nb\n{'c' * 37}")}
         self.assertEqual(
-            dmi.ProcessDmiSystemUuid.process("QPC_FORCE_POST_PROCESS", dependencies), ""
+            dmi.ProcessDmiSystemUuid.process("QPC_FORCE_POST_PROCESS", dependencies),
+            "b",
         )
```

Comment:
this one shows that the output is wrong IMO. it should be "" considering "b" is not a valid uuid 🤔

Reply:
See previous comment. I know this is a bad test, but I am not trying to reinvent legacy code or assumptions.

Reply:
This doesn't make me a happy developer, by the way, but I am setting limits on the scope of my work because I could spend all day/week/month/year improving bad old code like this.

Reply:
I see. I was concerned this test was reflecting some real world scenario.